Viraji patch 2

viraji opened the pull request:
Changed nav bar to:
Home | Licences | Community | Download | Press
from:
Home | About | Licences | Contribute | Contact Us
as per Issue #14 described in:
https://docs.google.com/document/d/1KVU44BA37tu-4DXYM99diCeqI6FeDiOFYQp5zLwl8vU/edit?usp=sharing

viraji committed:
Create README.md

viraji committed:
Update nav.html

viraji committed:
Create community.md

Adding new file for Community

viraji committed:
Add Press to nav bar

Move over content from Announcements:
https://github.com/thegooddocsproject/governance/wiki/Announcements

viraji committed:
Update Nav bar

Point Download link to:
https://github.com/thegooddocsproject/templates/releases/tag/v0.1

viraji committed:
Add weekly meeting minutes

Added a section on Weekly Meeting Minutes calling them from:
https://github.com/thegooddocsproject/governance/wiki/Weekly-Meetings

viraji committed:
Title case for all headings

Changed all headings to title case for consistency.

viraji committed:
Add new file for Roles

Content to be filled in later.

viraji committed:
Add new file for Decisions

Description copied off:
https://github.com/thegooddocsproject/governance/blob/master/ProjectSteeringCommittee.md

viraji committed:
Add new file for Decision Logs

Calling existing links.

viraji commented:
All changes in #14 done as per the agreed upon doc at:
https://docs.google.com/document/d/1KVU44BA37tu-4DXYM99diCeqI6FeDiOFYQp5zLwl8vU/edit?usp=sharing

Loquacity commented on contact.md:
```diff
@@ -22,15 +22,19 @@ The main channels are:
 * #random: Non-work-related stuff
 
-### Real-time conference calls
+### Real-Time Conference Calls
```
Is title case written down as our standard somewhere?

viraji replied:
> 
> Is title case written down as our standard somewhere?
Not sure, but all other headings were in title case except for these two that I changed - so I changed these as well to make the page consistent.

camerons replied:
We follow Google's style guide, which specifies sentence case for titles and headings: https://developers.google.com/style/capitalization#capitalization-in-titles-and-headings

Loquacity replied:
They'll all need to be updated to sentence case, then.

viraji replied:
Changed accordingly.

Loquacity commented on decisions.md:
```diff
@@ -0,0 +1,20 @@
+    -0: Mildly against, but mostly indifferent
+    -1 : Strongly against. This is expected to be accompanied by a reason.
+
+For most decisions, a motion is raised within one of TheGoodDocsProject open forums (slack, email list, etc) and those present vote and decide. Should a decision:
+
+    Be strategic or impact many within TheGoodDocsProject,
+    Or if there is contention in deciding, and after discussion at least one member is still strongly against the motion, by continuing to vote -1,
+
+Then the motion should be referred to the PSC.
```
```suggestion
For most decisions, a motion is raised within one of TheGoodDocsProject open forums (slack, email list, etc) and those present vote and decide. A decision should be referred to the PSC only if:

- The decision is strategic or could impact many people within TheGoodDocsProject
- If there is contention in deciding, and after discussion at least one member is still strongly against the motion, by continuing to vote -1
```

camerons replied:
We don't need to worry about fixing this lift-and-shift of the decisions page in this iteration.
It is about to get updated again based on https://docs.google.com/document/d/1hwdA2Gh3ePUFXk4juu7mSoAraxV-BDiarXaPCOY6K3w/edit

viraji replied:
Noted.

camerons commented on README.md:
```diff
@@ -0,0 +1,2 @@
+# thegooddocsproject.github.io
+Forking The Good Docs Project website
```
As you are doing a lift-and-shift, and this file doesn't contain any value. I'd suggest that we don't include this README.md file. (We can add it later.)

viraji replied:
Noted and readme.md deleted.

camerons commented on _includes/nav.html:
```diff
@@ -1,5 +1,5 @@
-  <a href="https://thegooddocsproject.dev/contribute.html" style="color:#ffffff;">Contribute</a> | 
-  <a href="https://thegooddocsproject.dev/contact.html" style="color:#ffffff;">Contact Us</a> | 
\ No newline at end of file
+  <a href="https://thegooddocsproject.dev/licences.html" style="color:#ffffff;">Licences</a>
```
Can we make this hyperlinks relative (licences.html) instead of absolute (https://thegooddocsproject.dev/licences.html)?
Doing so will mean we can test the site in a staging environment, or local environment.
Apply the same comment to other links too.

viraji replied:
Done.

mgan59 replied:
Relative links weren't applied, initially wouldn't work for preview/staging.  I made the adjustments.

camerons commented on community.md:
```diff
@@ -0,0 +1,11 @@
+
+
+
+*   [Contribute](https://thegooddocsproject.groups.io/g/contribute.md)
```
Have you checked these links in a rendered website? I haven't but suspect that contribute.html will find the correct page, but contribute.md will get a 404.

viraji replied:
Was unable to test in a rendered website, and so followed existing code. The link has now been changed accordingly.

mgan59 replied:
Tested these links, majority of them go to the groups.io as a `.md` link and don't load. I see there were new `.md` files added that match.  So I constructed local links.

camerons commented on contact.md:
```diff
@@ -22,15 +22,19 @@ The main channels are:
 We are nice to each other, in line with our [Code of Conduct](https://github.com/thegooddocsproject/governance/blob/master/CodeOfConduct.md).
 
-### Who are we?
+### Who are We?
```
We should use sentence-case as per https://developers.google.com/style/capitalization

camerons reviewed (changes requested):
I've added some comments.
I'm embarrassingly unskilled at setting up a local web server and hope to work that out soon. (Help welcomed).
I'm hoping someone can view these changes rendered in a web page and confirm they look okay and links work.

viraji committed:
Update contact.md

viraji committed:
Update community.md

Changed file extension from .md to .html.

viraji committed:
Delete README.md

Deleting the file following Cameron's comment: " As you are doing a lift-and-shift, and this file doesn't contain any value. I'd suggest that we don't include this README.md file. (We can add it later.)"

viraji reviewed (commented):
Review changes have now been attended to, please review the PR now.

mgan59 reviewed (approved):
Ran through these changes.  Sidenote, all pages were missing __front-matter__ which is required.  As a result the build only generate `markdown` in the `_site` folder as indicated in this [stackoverflow](https://stackoverflow.com/questions/58617116/jekyll-does-not-create-html-page-from-markdown-file).  I have no idea how this site builds/deploys without the front-matter.  I went through and added

```
---
layout: default
---
```

To all pages which means they now use the `_layout/default.html`

mgan59 committed:
fixed some links & added front-matter to pages

mgan59 commented:
There were conflicts with `index.md` from `master`.  I did a __REBASE__ with a `--force` push.  I'm assuming that these changes are final.  But if you need/want to continue working on this branch you will need to rebuild your local branch.

mgan59 commented:
> github has a limit 10 megs, so broke up the review gifs.

## Verification Top NavBar has relative links that work

![good-docs-review-pr-35-part1](https://user-images.githubusercontent.com/89339/90977246-eb1d6b80-e511-11ea-8439-c78b34febb8f.gif)

## Verification of Community Page -- bunch of additional relative links to new pages (Roles, Decision Logs)

![good-docs-pr-35-community](https://user-images.githubusercontent.com/89339/90977254-07210d00-e512-11ea-9b62-6cb420fe7aa9.gif)

mgan59 commented on Gemfile.lock:
```diff
@@ -8,7 +8,9 @@ GEM
     em-websocket (0.5.1)
```
Not sure how these changes crept in, I had stashed them... if we have a issue with the build I can undo them.

camerons commented:
Provided my approval based on screenshots from Morgan, review of code changes, and my trust of Morgan's review process.
(I don't have a local version to check.)

Loquacity commented:
:rocket:
